lib/msal-core/samples/VanillaJSTestApp/index.html

```diff
@@ -19,17 +19,12 @@
         </div>
     </div>
     <script>
-    /*var applicationConfig = {
-            clientID: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
-            authority: "https://login.microsoftonline.com/common",
-            graphScopes: ["user.read", "Mail.Send"],
-            graphEndpoint: "https://graph.microsoft.com/v1.0/me",
-            loginType: 'POPUP'
-    };*/
 
-    var msalConfig = {
+    // initialize MSAL
+    const msalConfig = {
         auth: {
             clientId: "Enter_your_client_id",
+            // clientId: "245e9392-c666-4d51-8f8a-bfd9e55b2456",
```

Comment:
do we want to leave this in here?

Reply:
I think so, its helpful to have a demo client id available.

Reply:
Is it an internal only client id?

Reply:
like is there concern around exposing to others

Reply:
Nope, its been in the repo for a while, hasn't it? And its only setup for localhost.

Reply:
Yes. I created this application (in the portal) just for demo purposes, so no issues in making this public. I left it alone here in the sample to have it ready for anyone to test it.